Makefile.in

```
@@ -4,7 +4,7 @@
 KEXT_NAME = NormalizInterface
 KEXT_SOURCES = src/normaliz.cc
 
-KEXT_CXXFLAGS = @NORMALIZ_CPPFLAGS@ -std=c++11
+KEXT_CXXFLAGS = @NORMALIZ_CPPFLAGS@ -std=gnu++11
```

Hmm, so the problem there was that sysjmp.h or so uses a GNU C extension, right? I am OK with adding a workaround to make sure things work with GAP 4.11.0 but. But I'd really prefer to fix the root cause on the GAP side, and make sure GAP headers are usable without GNU extensions. That shouldn't be too hard to fix thanks to our recent header restructuring -- at least on `master`, but hopefully we can achieve that on `stable-4.11`, too. I'll try to work on a PR for GAP later (have to take care of the kid right now, no babysitter today)

---

To be concrete, the problem is that with extensions turned on, Cygwin's gcc has siglongjmp and sigjmp_buf, so the configure script enables HAS_SIGSETJMP, which is used in system.h

However, then if we compile with extensions, the compiliation fails due to these things being missing.

---

OK, here is what's going on: the problem is not that GNU extensions are disabled, the problem is that POSIX extensions are *also* disabled. This suggests a way to fix this on the GAP side. Can you please try if this also works:

```suggestion
KEXT_CXXFLAGS = @NORMALIZ_CPPFLAGS@ -std=c++11 -D _POSIX_C_SOURCE=1
```

If it does, that would suggest a way to fix this in GAP: we just add something like this in `sysjmp.h` (resp. in `system.h`, for GAP 4.11) before the `#include <setjmp.h>`:
```C
#ifndef _POSIX_C_SOURCE
#define _POSIX_C_SOURCE 1
#endif
#include <setjmp.h>
```

---

Check everything over and this seems to fix the problem, and is the only problem in headers.

---

Unfortunately defining _POSIX_C_SOURCE like this causes a build failure for this package on FreeBSD. Luckily in 4.12 this will all be gone due to my restructuring of the GAP header files.

---

Due to this, I am going back to using `-std=gnu++11` (and once GAP 4.12 is out, we can just depend on that instead)